### PR TITLE
Added travis

### PR DESCRIPTION
**Pull request by delvedor**
Fixes https://github.com/fastify/benchmarks/issues/57

I've also configured the CI env.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+
+node_js:
+  - "lts/*"
```
Comment (mcollina) [inline .travis.yml]: This does not work as you'd think. Can you please put Node 8 there?

Comment (delvedor) [reply .travis.yml]: I've followed [this](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions) guide.

Comment (mcollina) [reply .travis.yml]: That guide is not correct and it does not produce stable results. When we tag a new lts major version in Node.js, the CI will stop testing on the previous LTS and start testing the new.

**Comment by mcollina** [comment]
Thanks!

**Pull request merged**